ldap_mysql_granter/email_tool.py

```diff
@@ -127,7 +127,7 @@ def sendChangePasswordInvite(self, mysqlClusters, mysqlUserAtHost,
         message = util.renderTemplate("password_change_invite.tmpl",
                                       templateArgDict)
         logger.info("Sent ChangePassInvite to [%s] for %s on %s with password %s",
-                    ', '.join(receipients), mysqlUser, ",".join(mysqlClusters), mysqlPass)
+                    receipients, mysqlUser, mysqlClusters, mysqlPass)
```

Comment:
I know you didn't do it, but would you be able to fix the spelling of `recipients` here and everywhere else? :)

Reply:
Yes.